packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -17,6 +16,8 @@ import 'mac.dart';
 
 IOSWorkflow get iosWorkflow => context.putIfAbsent(IOSWorkflow, () => new IOSWorkflow());
 
+IMobileDevice get iMobileDevice => IMobileDevice.instance;
```

Comment:
Could move this to `mac.dart` and kill the `instance` getter there. If we do that, probably worth doing the same for the `Xcode.instance` getter.